kernel/src/actions/mod.rs

```diff
@@ -613,6 +613,9 @@ pub(crate) struct CommitInfo {
     pub(crate) kernel_version: Option<String>,
     /// A place for the engine to store additional metadata associated with this commit
     pub(crate) engine_info: Option<String>,
+    /// A unique transaction identified for this commit. When the `catalogManaged` table feature is
+    /// enabled (not yet implemented), this field will be required.
```

Comment:
```suggestion
    /// A unique transaction identified for this commit. When the `catalogManaged` table feature is
    /// enabled (not yet implemented), this field will be required. Otherwise, it is optional.
```
(I believe delta-spark always writes it out, and has for several years)

Reply:
thanks! tracking in follow up https://github.com/delta-io/delta-kernel-rs/issues/1149